napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -64,6 +68,55 @@ def store_widget(self, *args, **kwargs):
     monkeypatch.setattr(_QtMainWindow, "current", mock_current_main_window)
 
 
+@dataclass
+class ShowStatus:
+    show_notification_count: int = 0
+    show_traceback_count: int = 0
+
+
+@pytest.fixture(autouse=True)
+def raise_on_show(monkeypatch, qtbot):
+    def _raise_on_show(self, *args, **kwargs):
+        raise RuntimeError('error!')
+
+    monkeypatch.setattr(NapariQtNotification, 'show', _raise_on_show)
+    monkeypatch.setattr(TracebackDialog, 'show', _raise_on_show)
```

Comment:
This fixture raises an exception on the show call if another fixture does not mock it.